src/patternfly/components/Nav/nav.scss

```diff
@@ -1,169 +1,111 @@
-  --pf-c-nav__horizontal-list-link--active--Color: var(--pf-global--active-color--400);
-  --pf-c-nav__horizontal-list-link--focus--Color: var(--pf-global--active-color--400);
-
-  // after
-  --pf-c-nav__horizontal-list-link--after--Height: #{pf-size-prem(3px)};
-  --pf-c-nav__horizontal-list-link--hover--after--BackgroundColor: var(--pf-global--active-color--400);
-  --pf-c-nav__horizontal-list-link--active--after--BackgroundColor: var(--pf-global--active-color--400);
-  --pf-c-nav__horizontal-list-link--focus--after--BackgroundColor: var(--pf-global--active-color--400);
-
-  // List tertiary item
-  --pf-c-nav__tertiary-list-item--MarginRight: var(--pf-global--spacer--xl);
-  --pf-c-nav__tertiary-list-link--PaddingTop: var(--pf-global--spacer--md);
-  --pf-c-nav__tertiary-list-link--PaddingBottom: var(--pf-global--spacer--sm);
-
-  // font
-  --pf-c-nav__tertiary-list-link--FontWeight: var(--pf-global--FontWeight--normal);
-
-  // color
-  --pf-c-nav__tertiary-list-link--Color: var(--pf-global--Color--dark-100);
-  --pf-c-nav__tertiary-list-link--hover--Color: var(--pf-global--link--Color);
-  --pf-c-nav__tertiary-list-link--active--Color: var(--pf-global--link--Color);
-  --pf-c-nav__tertiary-list-link--focus--Color: var(--pf-global--link--Color);
-
-  // after
-  --pf-c-nav__tertiary-list-link--after--Height: #{pf-size-prem(3px)};
-  --pf-c-nav__tertiary-list-link--hover--after--BackgroundColor: var(--pf-global--link--Color);
-  --pf-c-nav__tertiary-list-link--active--after--BackgroundColor: var(--pf-global--link--Color);
-  --pf-c-nav__tertiary-list-link--focus--after--BackgroundColor: var(--pf-global--link--Color);
-
-  // Subnav links
-  --pf-c-nav__subnav--MarginTop: var(--pf-global--spacer--sm);
+  // Link
+  --pf-c-nav__link--FontSize: var(--pf-global--FontSize--md);
+  --pf-c-nav__link--FontWeight: var(--pf-global--FontWeight--normal);
+  --pf-c-nav__link--PaddingTop: var(--pf-global--spacer--md);
+  --pf-c-nav__link--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-nav__link--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-nav__link--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-nav__link--md--PaddingRight: var(--pf-global--spacer--lg);
+  --pf-c-nav__link--md--PaddingLeft: var(--pf-global--spacer--lg);
+  --pf-c-nav__link--Color: var(--pf-global--Color--dark-100);
+  --pf-c-nav__link--hover--Color: var(--pf-global--Color--dark-100);
+  --pf-c-nav__link--active--Color: var(--pf-global--Color--dark-100);
+  --pf-c-nav__link--focus--Color: var(--pf-global--Color--dark-100);
+  --pf-c-nav__link--disabled--Color: var(--pf-global--disabled-color--100);
+  --pf-c-nav__link--BackgroundColor: transparent;
+  --pf-c-nav__link--hover--BackgroundColor: var(--pf-global--BackgroundColor--light-300);
+  --pf-c-nav__link--active--BackgroundColor: var(--pf-c-nav__link--hover--BackgroundColor);
+  --pf-c-nav__link--focus--BackgroundColor: var(--pf-c-nav__link--hover--BackgroundColor);
```

Comment:
Same here - if we're going to have a var for each, we should probably not create the dependency between them if one changes.